docs/ref/migration-operations.txt

```diff
@@ -185,15 +185,21 @@ defaults and always applies them in the Django ORM code.
 ``RemoveField``
 ---------------
 
-.. class:: RemoveField(model_name, name)
+.. class:: RemoveField(model_name, name, default=NOT_PROVIDED)
+
+.. versionchanged:: 5.1
```

Comment:
```suggestion
.. versionchanged:: 5.2
```
(though this may end up in 6.0)
I would also put the versionchanged at the bottom of the class declaration (to not disrupt the flow)